angular.json

```diff
@@ -97,6 +97,7 @@
     }
   },
   "cli": {
-    "schematicCollections": ["@angular-eslint/schematics"]
+    "schematicCollections": ["@angular-eslint/schematics"],
+    "analytics": "4e54d43f-6f1b-438d-aadf-bdee1605cc24"
```

Comment:
What is this for?

Reply:
Angular asks me if we agree in opting into their analytics. I just said yes to clear the message (Angular asks it on every command) and help them collect telemetry as a fellow library developer. :slightly_smiling_face: 

![image](https://github.com/user-attachments/assets/31493524-6504-4c46-86f2-f9a35a89adb7)

Reply:
Oh, okay :smile: